client/jetpack-connect/colors.scss

```diff
@@ -38,4 +38,22 @@
 		color: var( --color-white );
 		background: var( --color-accent );
 	}
+
+	// @todo Once Calypso has updated to color-studio v2, WooCommerce Onboarding's Connection flow can switch to using studio variables.
```

Comment:
Did you consider upgrading the package? Were there blockers with that?

Reply:
Sounds totally reasonable, thanks for the detail 🙂

Reply:
Just FYI: I’m updating Color Studio in #35949 and will take care of all these values.